### PR TITLE
Add showit

### PR DESCRIPTION
**Pull request by joshmoore**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

A prerequisite of gh-10457

### CODE REVIEW

File: recipes/starfish/showit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  name: showit
+  version: {{ SHOWIT_VERSION }}
+
+source:
+  url:  https://github.com/freeman-lab/showit/archive/ef76425797c71fbe3795b4302c49ab5be6b0bacb.zip
+  sha1: 64484b77fba4fb641daa964ff28b6cb151810222
+
+build:
+  noarch: python
+  number: 1
```
Comment (bgruening) [inline recipes/starfish/showit/meta.yaml]: please start with 0

File: recipes/starfish/showit/build.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd ./lib/python
```
Comment (bgruening) [inline recipes/starfish/showit/build.sh]: please use popd and pushd to change directories

File: recipes/starfish/showit/build.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (bgruening) [inline recipes/starfish/showit/build.sh]: Showit should move up one directory level I think.

**Comment by joshmoore** [comment]
@bgruening : definitely, thanks for all the pointers. Turns out I was struggling with the `cd lib/python` step that had been created by skeleton(?).

File: recipes/showit/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  number: 0
+
+requirements:
+  host:
+    - setuptools
+    - python
+  run:
+    - python
+    - matplotlib
+    - numpy !=1.13.0
```
Comment (bgruening) [inline recipes/showit/meta.yaml]: last question, this numpy version is not allowed but any later one is ok?

Comment (joshmoore) [reply recipes/showit/meta.yaml]: This is from their requirements.txt file, due to the version in travis leading to:

> TypeError: numpy boolean subtract, the `-` operator, is deprecated, use the bitwise_xor, the `^` operator, or the logical_xor function instead.

See https://github.com/spacetx/starfish/pull/17

**Comment by joshmoore** [comment]
Re-opening due to:

```
ssh: connect to host github.com port 22: Operation timed out
```

**Comment by joshmoore** [comment]
Re-testing

**Comment by bgruening** [comment]
restarted

File: recipes/showit/meta.yaml (hunk 4 above)
Comment (bgruening) [reply recipes/showit/meta.yaml]: I read it like only <1.13 is supported as the operator is deprecated :(

**Comment by bgruening** [comment]
Green!

File: recipes/showit/meta.yaml (hunk 4 above)
Comment (ambrosejcarr) [reply recipes/showit/meta.yaml]: We only intended to disallow that particular version.

Comment (bgruening) [reply recipes/showit/meta.yaml]: @ambrosejcarr numpy 1.13.1 does support the operator again?

**Comment by bgruening** [comment]
Ok, upstream confirmed. Good to go!

**Pull request merged**